perllib/Integrations/Bartec.pm

```diff
@@ -251,6 +251,18 @@ sub Jobs_FeatureScheduleDates_Get {
     return [ values %$out ];
 }
 
+sub relevant_jobs {
+    my ($self, $uprn, $schedules) = @_;
+    my $jobs = $self->Jobs_FeatureScheduleDates_Get($uprn);
+    my %schedules = map { $_->{JobName} => $_ } @$schedules;
+    my @jobs = grep {
+        my $name = $_->{JobName};
+        $schedules{$name}->{Feature}->{Status}->{Name} eq 'IN SERVICE'
+        && $schedules{$name}->{Feature}->{FeatureType}->{ID} != 6815;
```

Comment:
This Peterborough-specific magic number feels a bit out of place here. Would this method be better in the `Peterborough.pm` cobrand module? Or at least a comment here explaining what it's doing and how it might need refactoring for the next Bartec integration.

Reply:
I actually had this function in Peterborough.pm originally but moved it to make it easier to not pass $bartec around, probably not a good basis for that decision :) Will move it back.